src/modules/previewpane/MonacoPreviewHandler/FileHandler.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+
+namespace Microsoft.PowerToys.PreviewHandler.Monaco
+{
+    public static class FileHandler
+    {
+        /// <summary>
+        /// Converts a file extension to a language monaco id.
+        /// </summary>
+        /// <param name="fileExtension">The extension of the file (without the dot).</param>
+        /// <returns>The monaco language id</returns>
+        public static string GetLanguage(string fileExtension)
+        {
+            try
+            {
+                JsonDocument a = JsonDocument.Parse(File.ReadAllText(Settings.AssemblyDirectory + "\\languages.json"));
```

Comment:
nit: `a` not a very descriptive var name..

Reply:
Forgot this placeholder 😅